packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -0,0 +1,40 @@
+name: rancher-alerting-drivers
+home: ""
+sources: []
+version: 1.0.1
+description: The manager for third-party webhook receivers used in the Alertmanger
+keywords:
+- monitoring
+- alertmanger
+- webhook
+maintainers: []
+icon: ""
+apiversion: v2
+condition: ""
+tags: ""
+appversion: 1.16.0
+deprecated: false
+annotations:
+  catalog.cattle.io/certified: rancher
```

Comment:
Does this need `catalog.cattle.io/namespace: cattle-monitoring-system` or was the intention to remove it since that prevents this chart from being installed in any other namespace?

Reply:
IT is removed to allow users to install it to any namespace.

Reply:
With it you could only ever install one per cluster, which negates the "per project" approach